examples/workbooks/LeadsExample.ts

```diff
@@ -0,0 +1,325 @@
+import * as FF from "@flatfile/configure";
+import { FlatfileRecord } from "@flatfile/hooks";
+
+/*
+ * Types
+ */
+
+type Nil = null | undefined;
+
+type Falsy = null | undefined | false | "" | 0;
+
+/*
+ * Guards
+ */
+
+/**
+ * Helper function to determine if a value is null.
+ * Useful in if/else statements or ternaries.
+ *
+ * @param {*} x - Any object/value
+ *
+ * @example
+ * if (isNull(x)) {
+ *   ...
+ * } else {
+ *   ...
+ * }
+ */
+export const isNull = (x: unknown): x is null => x === null;
+
+/**
+ * Helper function to determine if a value is undefined.
+ * Useful in if/else statements or ternaries.
+ *
+ * @param {*} x - Any object/value
+ *
+ * @example
+ * if (isUndefined(x)) {
+ *   ...
+ * } else {
+ *   ...
+ * }
+ */
+export const isUndefined = (x: unknown): x is undefined => x === undefined;
```

Comment:
Can you replace these helper functions with lodash equivalents.  and a link to lodash.  I think that's a reasonable medium ground.

Reply:
So I read through the Lodash docs and the code... Not impressed with their lack of type safety (pretty sure that's partly why Ramda was created). BUT, I've done something very specific my `isNil` fn -- it also considers an empty string null -- that allows me to write more concise and clear code (arguably) with regards to FF values. Still thinking on this though...